vcr/interface.go

```diff
@@ -68,34 +69,38 @@ type Writer interface {
 	StoreRevocation(r credential.Revocation) error
 }
 
+// Resolver binds all read type of operations into an interface
+type Resolver interface {
+	// Registry returns the concept registry
+	Registry() concept.Registry
+	// Search for matching credentials based upon a query. It returns an empty list if no matches have been found.
+	Search(query concept.Query) ([]vc.VerifiableCredential, error)
+	// Verify checks if a credential is valid and trusted at the given time.
+	// The time check is optional, so credentials can be issued that will become valid.
+	// If valid no error is returned.
+	Verify(vcToVerify vc.VerifiableCredential, at *time.Time) error
```

Comment:
I feel like `Verify` shouldn't be a member of `Resolver`. Does something break when you move it back to `VCR`?

Reply:
nope